src/components/Pivot/Pivot.md

```diff
@@ -0,0 +1,43 @@
+# Pivot
+Includes a Persona component with contact options.
```

Comment:
This should say something more along the lines of:
"A layout component that allows a user to switch between different sets of content or filters on the content currently visible. This is in contrast to a purely navigational UI element such as NavBar or a commanding element such as CommandBar. An example would be Read vs. Unread items in the Outlook Web App."